user_docs/en/changes.t2t

```diff
@@ -50,10 +50,8 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
     -
   -
 - Sound split: (#12985, @mltony)
-  - Allows splitting NVDA sounds into one channel (e.g. left) while sounds from all other applications are placed in the other channel (e.g. right).
+  - Allows splitting NVDA sounds in one channel (e.g. left) while sounds from all other applications in the other channel (e.g. right).

```

Comment:
Although this is what we had before, the PR being reverted fixed the phrasing. Can we skip reverting this bit?

Reply:
```suggestion
  - Allows splitting NVDA sounds into one channel (e.g. left) while sounds from all other applications are directed to the other channel (e.g. right).
```